suseRegisterInfo/suseRegisterInfo.spec

```
@@ -15,13 +15,13 @@
 # Please submit bugfixes or comments via https://bugs.opensuse.org/
 #
 
-
-%{!?python_sitelib: %global python_sitelib %(%{__python} -c "from distutils.sysconfig import get_python_lib; print get_python_lib()")}
-%if 0%{?fedora} || 0%{?suse_version} > 1320
+%if 0%{?fedora} || 0%{?suse_version} > 1320 || 0%{?rhel}
```

This package is installed on Clients. So we need to support also older RHEL versions.
I think python3 as default is only available since RHEL8, right?
Even older version of RHEL7 only have python2.

So maybe:

```suggestion
%if 0%{?fedora} || 0%{?suse_version} > 1320 || 0%{?rhel} >= 8
```